src/mumble/Settings.cpp

```diff
@@ -1017,8 +1017,7 @@ void Settings::load(QSettings *settings_ptr) {
 	settings_ptr->endGroup();
 }
 
-#undef SAVELOAD
-#define SAVELOAD(var, name)                               \
+#define SAVE(var, name)                               \
```

Comment:
```suggestion
#undef LOAD
#define SAVE(var, name)                               \
```
This way you can't accidentally load in the `save` function (aka: prevents copy&paste errors)

Reply:
Very smart :)